style.css

```diff
@@ -655,3 +694,343 @@ label {
   width: 90px;
   text-align: right;
 }
+
+.link-button:hover,
+.link-button2:hover,
+.resume:hover,
+.sub-bot:hover {
+  color: blue;
+  background: wheat;
+  border: blue;
+}
+
+@media screen and (min-width: 768px) {
+  main {
+    width: 768px;
+    align-items: center;
+  }
+
+  body {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+  }
+
+  .page1 {
+    background-image: url("/img/desktop-ver.png");
+    background-repeat: no-repeat;
```

Comment:
- [ ] Dear @mohisa302. Although you add an image I can see you used a '/' to refer to the source. This will only work for a local repository. Please instead use `./` to refer to a relative path. 
> EG.  `src="/img/env.png"` to `src="./img/env.png"`
- [ ] Also, As You can see your design can use more styled to align with the Figma design. This includes the display of different screen sizes, colors, and background decorations. Please make sure to add more style to better design your `Header/Headline` section.

### Your design on a `1000px` screen size
![Untitled](https://user-images.githubusercontent.com/69077061/206499711-f78a709d-161e-49e7-98ed-6850edd8d626.png)

### Figma design 
![Screenshot 2022-12-07 203211](https://user-images.githubusercontent.com/69077061/206501590-706459c7-4a13-4d2b-b6a4-b9fffec8414d.png)